src/main/scala/com/lucidchart/open/relate/SqlStatementParser.scala

```diff
@@ -84,14 +84,7 @@ private[relate] object SqlStatementParser {
    * @param query the current query in a StringBuilder
    */
   private def insertCommaSeparated(count: Int, query: StringBuilder): Unit = {
-    if (count > 0) {
-      query += '?'
-    }
-    var i = 1
-    while (i < count) {
-      query.append(",?")
-      i += 1
-    }
+    query append Seq.fill(count)("?").mkString(",")
```

Comment:
This will likely be much less efficient than the old code.

It could probably be improved by using:

```scala
Iterator.fill(count)('?').addString(query, ",")
```